File: 1-Iniciante/JavaScript/1013.js
let [A, B, C] = lines.shift().split(" ").map((x) => parseInt(x));

let maiorAB = ((A+B+(Math.abs(A-B)))/2);

let maior = ((maiorAB+C+(Math.abs(maiorAB-C)))/2);

console.log(`${maior} eh o maior`);

File: 1-Iniciante/Java/1013.java
package beecrowdjava1013;

import java.io.*;
import java.util.*;

public class BeeCrowdJava1013 {
  
    public static void main(String[] args) throws IOException{
        
      Scanner in = new Scanner(System.in);
      
       int A, B, C, resposta, absoluto;
       A = in.nextInt();
       B = in.nextInt();
       C = in.nextInt();
       absoluto = (int) (((A+B + Math.abs(A-B))/ 2));
       resposta = (absoluto > C)?absoluto:C; 
       System.out.println(resposta + " eh o maior");
       System.out.println(((Object)absoluto).getClass().getSimpleName());
    
    }
    
}

